src/components/HomeSectionClients/index.tsx

```diff
@@ -9,235 +9,228 @@ import {
-      left={
+      <Content>
         <LogoText>
           <FramerLogo />
           <p>
             <a href="https://framer.com/">Framer</a> is a web-based tool for
             interactive interface design – one of the hottest ones, today.
           </p>
+          <TagContainer>
+            <div>#react-performance</div>
+            <div>#build-performance</div>
+            <div>#core-web-vitals</div>
+          </TagContainer>
         </LogoText>
-      }
-      right={
-        <Content>
+        <p>
+          We worked with Framer to perfect their user experience – by making
+          user interactions smoother and app loading faster.
+        </p>
+        <p>
+          By working together,{' '}
+          <Mark>
+            we reduced Speed Index and First CPU Idle of Framer Web by 40-45%
+          </Mark>
+          . We also set up tooling around runtime perf – to help Framer keep
+          &amp; maintain achieved results in the future.
+        </p>
+        <blockquote>
           <p>
-            We worked with Framer to perfect their user experience – by making
-            user interactions smoother and app loading faster.
-          </p>
-          <p>
-            By working together,{' '}
+            We’ve been very satisfied by working with Ivan! Ivan helped us to
+            improve loading performance, runtime performance, and tooling around
+            it.{' '}
             <Mark>
-              we reduced Speed Index and First CPU Idle of Framer Web by 40-45%
+              The quality of his work, approach, documentation, etc. has been
+              outstanding.
             </Mark>
-            . We also set up tooling around runtime perf – to help Framer keep
-            &amp; maintain achieved results in the future.
           </p>
-          <blockquote>
-            <p>
-              We’ve been very satisfied by working with Ivan! Ivan helped us to
-              improve loading performance, runtime performance, and tooling
-              around it.{' '}
-              <Mark>
-                The quality of his work, approach, documentation, etc. has been
-                outstanding.
-              </Mark>
-            </p>
-            <footer>
-              <a href="https://twitter.com/eelco">Eelco Lempsink</a>, VP of
-              Engineering @ Framer
-            </footer>
-          </blockquote>
-        </Content>
-      }
-    />
+          <footer>
+            <a href="https://twitter.com/eelco">Eelco Lempsink</a>, VP of
+            Engineering @ Framer
+          </footer>
+        </blockquote>
+      </Content>
 
-    <HomeLeftRightWrapper
-      left={
+      <Content>
         <LogoText>
           <CastorLogo />
           <p>
             <a href="https://www.castoredc.com/">Castor</a> is a tech platform
             that makes clinical trials easier.
           </p>
+          <TagContainer>
+            <div>#react-performance</div>
+            <div>#core-web-vitals</div>
```

Comment:
Also: playing with the layout, I noticed that I really want to click a hashtag as if it were a button – but it wasn’t clickable. This may be annoying to visitors who try to click these tags.

How about we change them to not look like hashtags? Eg `#react-performance` → `react performance` (all lower case) and so on